core/templates/dev/head/services/ContextService.js

```diff
@@ -100,8 +109,29 @@ oppia.factory('ExplorationContextService', [
           }
 
           throw Error(
-            'ERROR: ExplorationContextService should not be used outside the ' +
-            'context of an exploration.');
+            'ERROR: ContextService should not be used outside the ' +
+            'context of an exploration or a question.');
+        }
+      },
+
+      // Returns a string representing the questionId (obtained from the
+      // URL).
+      getQuestionId: function() {
+        if (questionId) {
+          return questionId;
+        } else {
```

Comment:
There should be some check here that we're in a QUESTION_EDITOR context, otherwise throw an error.

Reply:
Error message: "Error: You are not in Question Context." sounds good?

Reply:
Same for getExplorationId() as well?